pkg/reconciler/reconciler.go

```diff
@@ -237,12 +237,12 @@ func WithOverrideValues(overrides map[string]string) Option {
 	}
 }
 
-// WithDependentWatchesEnabled is an Option that configures whether the
+// WithSkipDependentWatches is an Option that configures whether the
 // Reconciler will register watches for dependent objects in releases and
 // trigger reconciliations when they change.
 //
 // By default, dependent watches are enabled.
-func SkipDependentWatches(skip bool) Option {
+func WithSkipDependentWatches(skip bool) Option {
```

Comment:
@joelanford this was the only code change made here. 
The rest are just comments.